lib/mayaHydra/flowViewportAPIExamples/footPrintNode/CMakeLists.txt

```
@@ -0,0 +1,102 @@
+set(TARGET_NAME mayaHydraFootPrintNode)
+
+add_library(${TARGET_NAME} SHARED)
+
+# -----------------------------------------------------------------------------
+# sources
+# -----------------------------------------------------------------------------
+target_sources(${TARGET_NAME} 
+    PRIVATE
+        mhFootPrintNode.cpp
+)
+
+set(HEADERS
+	PRIVATE
+)
+
+# -----------------------------------------------------------------------------
+# compiler configuration
+# -----------------------------------------------------------------------------
+target_compile_definitions(${TARGET_NAME}
+    PRIVATE 
+        # M3dView needs this (technically, MNativeWindowHdl.h)
+        # private atm because M3dView is only used in .cpp right now
+        # gcc will NOT set this automatically
+        $<$<BOOL:${IS_LINUX}>:LINUX>
+        # Not sure if msvcc sets this automatically, but won't hurt to redefine
+        $<$<BOOL:${IS_WINDOWS}>:_WIN32>
+		MFB_ALT_PACKAGE_NAME="${TARGET_NAME}" # This is used by Pixar HdSceneIndexPluginRegistry::Define to create a unique name
+)
+
+# -----------------------------------------------------------------------------
+# include directories
+# -----------------------------------------------------------------------------
+if(DEFINED MAYAUSD_VERSION)
+    target_compile_definitions(${TARGET_NAME}
+        PRIVATE
+            MAYAUSD_VERSION=${MAYAUSD_VERSION}
+    )
+endif()
+
+if(DEFINED MAYAHYDRA_VERSION)
```

This should not be optional, and is defined unconditionally in the top-level CMakeLists.txt.  Therefore, I suggest:
```suggestion
if(NOT DEFINED MAYAHYDRA_VERSION)
    message(FATAL_ERROR "No MAYAHYDRA_VERSION defined, aborting configuration." )
endif()
```